web/looking-good/.eslintrc.js

```diff
@@ -102,10 +102,6 @@ module.exports = {
     'react-hooks/rules-of-hooks': 'error',
     'react-hooks/exhaustive-deps': 'error',
 
-    'sort-imports-es6-autofix/sort-imports-es6': [2, {
-      'ignoreCase': false,
-      'ignoreMemberSort': false,
-      'memberSyntaxSortOrder': ['all', 'single', 'multiple', 'none']
-    }],
+    'sort-imports-es6-autofix/sort-imports-es6': [0],
```

Comment:
Why did you change this?

Reply:
Oh whoops, I was having trouble getting my imports in the order the linter wanted, I intended to put this back, will do so!